File: ps_java/src/algorithm/boj/sort/BOJ10989.java
package algorithm.boj.sort;

import java.io.*;

public class BOJ10989 {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));
        int n = Integer.parseInt(br.readLine());
        int[] arr = new int[10001];
        for (int i = 0; i < n; i++) {
            arr[Integer.parseInt(br.readLine())]++;
        }
        for (int i = 0; i < arr.length; i++) {
            if(arr[i] > 0)
                for(int j = arr[i]-1; j >= 0; j--) {
                    bw.write(String.valueOf(i) + "\n");
                }
        }
        bw.flush();
    }
}

File: ps_kotlin/src/main/kotlin/boj/boj10989.kt
package boj

fun main() {
    val br = System.`in`.bufferedReader()
    val bw = System.out.bufferedWriter()

    val n = br.readLine().toInt()
    val arr = IntArray(20001)
    for (i in 0 until n) {
        arr[br.readLine().toInt() + 10000]++
    }
    for ((i, b) in arr.withIndex()) {
        if (b != 0) {
            repeat(b) {
                bw.write("${i - 10000}\n")
            }
        }
    }
    bw.flush()
}